src/components/RequestListEntry.svelte

```diff
@@ -9,7 +9,7 @@ export let request;
 $: user = request.created_by || {}
 $: size = request.size
 $: from = request.origin && request.origin.description
-$: to = request.meeting ? request.meeting.name : request.destination.description
+$: to = request.destination.description
```

Comment:
```suggestion
$: to = request.destination?.description
```
Not sure if there is really a case where you will get a request without destination or description